### DISCUSSION TITLE
Receiving e-mails on behalf of

### DISCUSSION THREAD

**Discussion by nicky-lopes** [postalserver/postal#1539] [Help with using Postal]
Hello,

I configured the postal and everything is OK but I'm receiving e-mails on my Outlook like this:
![onbehalf](https://user-images.githubusercontent.com/50456353/129029419-9ac3faed-ccaa-4fb8-821c-13eaccab5887.png)
The e-mails are coming "on behalf of" and the sender is the return path (rp.domain.com.br)

how and where can I configure this the receive those e-mails without changing the sender?

I tried to unable to send from any sender like the post below but it didn't fix 
https://github.com/postalserver/postal/issues/168#issuecomment-302261864

Maybe something I missing.

thanks in advance!

**Comment by willpower232** [comment]
how are you sending the messages?

**Reply by nicky-lopes** [reply]
I'm sending through Outlook using the SMTP user and key

**Reply by willpower232** [reply]
Postal is not designed for use with a desktop email client so that might explain it.

**Reply by nicky-lopes** [reply]
OK, but I got another server with an old version of the postal and when I configured to send an e-mail through the outlook I received with the e-mail sender and not "on behalf of".
Even when I try to send through API in a website it comes "on behalf of". I configured on the Outlook because its easier to make tests.

This is the header of the message that I receive from the new version:
I try to send with the domain.com.br but I receive rp.domain.com.br on behalf of domain.com.br

Resent-Sender: <bcu2wz@rp.domain.com.br>
X-Postal-MsgID: AUVQ3jWoTn5k
Received: from NOTENGRENATO (::ffff:1.0.0.2 [::ffff:1.0.0.2]) by postal02.domain.com.br with SMTP; Wed, 11 Aug 2021 19:03:10 -0000
From: Renato <teste2@domain.com.br>
To: <person1@gmail.com>
CC: Renato <renato@otherdomain.com.br>
Subject: reerrererer
Date: Wed, 11 Aug 2021 16:03:09 -0300
Message-ID: <00c701d78ee3$86e8e7d0$94bab770$@cloud-mail.com.br>
MIME-Version: 1.0
Content-Type: multipart/alternative;
    boundary="----=_NextPart_000_00C8_01D78ECA.61AA55D0"
X-Mailer: Microsoft Outlook 16.0
Thread-Index: AdeO44YMdNoFyJNfTc2oTdTP0z/r1A==
Content-Language: en-us
Resent-Message-ID: <4GlK323ztGzYm3bf@c1f1.mail.com>
Resent-Date: Wed, 11 Aug 2021 16:03:10 -0300
Resent-From: <bcu2wz@rp.domain.com.br>
X-Sender: bcu2wz@rp.domain.com.br
Sender: <bcu2wz@rp.domain.com.br>
X-Envelope-From: bcu2wz@rp.domain.com.br
Return-Path: bcu2wz@rp.domain.com.br
X-MS-Exchange-Organization-Network-Message-Id: cc3cceb3-3fdb-49f9-4568-08d95cfabb60
X-MS-Exchange-Organization-AVStamp-Enterprise: 1.0
X-MS-Exchange-Organization-AuthSource: SKYSRV003.mail.net.br
X-MS-Exchange-Organization-AuthAs: Anonymous
X-MS-Exchange-Transport-EndToEndLatency: 00:00:00.1572351
X-MS-Exchange-Processed-By-BccFoldering: 15.01.2106.013

--------------------------------------------------

This is the Header when I use the old version

Resent-Sender: <rsqkfy@rp.domain.com.br>
X-Postal-MsgID: jKxcADVjstPp
Received: from NOTENGRENATO (::ffff:1.0.0.2 [::ffff:186.193.239.2]) by domain.com.br with SMTP; Thu, 12 Aug 2021 11:41:50 -0000
From: Renato Germano <teste2@domain.com.br>
To: <person1@gmail.com>
CC: Renato <renato@domain.com.br>
Subject: Teste Postal V1
Date: Thu, 12 Aug 2021 08:41:50 -0300
Message-ID: <000501d78f6f$0a21d450$1e657cf0$@domain.com.br>
MIME-Version: 1.0
Content-Type: multipart/alternative;
    boundary="----=_NextPart_000_0006_01D78F55.E4DADDE0"
X-Mailer: Microsoft Outlook 16.0
Thread-Index: AdePbwLEo9G3gX7TTw6wXHlWkK2q8w==
Content-Language: en-us
Resent-Message-ID: <4GllCL6PnczYl2lR@c1f4.cloud-mail.com>
Resent-Date: Thu, 12 Aug 2021 08:41:50 -0300
Resent-From: <rsqkfy@rp.domain.com.br>
Return-Path: rsqkfy@rp.domain.com.br
X-MS-Exchange-Organization-Network-Message-Id: 8ff09173-585f-423b-680c-08d95d863038
X-MS-Exchange-Organization-AVStamp-Enterprise: 1.0
X-MS-Exchange-Organization-AuthSource: SKYSRV003.mail.net.br
X-MS-Exchange-Organization-AuthAs: Anonymous
X-MS-Exchange-Transport-EndToEndLatency: 00:00:00.1622242
X-MS-Exchange-Processed-By-BccFoldering: 15.01.2106.013
 
-----------------------------------------

The only difference is that the new server is adding this lines

X-Sender: bcu2wz@rp.domain.com.br
Sender: <bcu2wz@rp.domain.com.br>
X-Envelope-From: bcu2wz@rp.domain.com.br

**Reply by nicky-lopes** [reply]
We identified that our e-mail provisor might be adding this lines when we send an e-mail thought the new server.
I'll talk to them first, and if it's not a problem with them I will come back later.

Thanks for to help.
